applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -105,9 +119,21 @@ static void stream_enabled_cb(struct bt_audio_stream *stream)
 {
 	int ret;
 
-	ret = bt_audio_stream_start(stream);
-	if (ret) {
-		LOG_ERR("Unable to start stream: %d", ret);
+	if (stream->conn == headset_conn[AUDIO_CHANNEL_LEFT]) {
+		ret = bt_audio_stream_start(&audio_streams[AUDIO_CHANNEL_LEFT]);
+		while (ret != 0) {
```

Comment:
Is this mechanism needed? If there is something wrong that causes bt_audio_stream_start to fail it will be stuck in this loop

Reply:
Yes, this is because the host should finish the establishment of first CIS ISO and then start the second one, because the controller cannot handle CIS ISO channel in parallel. Otherwise only one CIS will be connected properly and the other one will fail.
An issue has been created and tracked in here:
https://github.com/zephyrproject-rtos/zephyr/issues/46829

Reply:
Maybe implement a counter that breaks the loop after N tries?

Reply:
There's an open PR for related feature:
https://github.com/zephyrproject-rtos/zephyr/pull/47549